From 9ecc7269de463904fe80fabcb1aa36a04603013b Mon Sep 17 00:00:00 2001
From: Daniel van Vugt <daniel.van.vugt@canonical.com>
Date: Wed, 12 May 2021 18:15:00 +0800
Subject: [PATCH] theme: Remove background-color and box-shadow from
 workspace-background

The first is completely invisible and the second is barely visible, but
together they were incurring almost 40% of the render time during
overview animations. That's not a good deal so just remove them.
---
 data/theme/gnome-shell-sass/widgets/_window-picker.scss | 2 --
 1 file changed, 2 deletions(-)

diff --git a/data/theme/gnome-shell-sass/widgets/_window-picker.scss b/data/theme/gnome-shell-sass/widgets/_window-picker.scss
index b3b93f291a..afa8de036d 100644
--- a/data/theme/gnome-shell-sass/widgets/_window-picker.scss
+++ b/data/theme/gnome-shell-sass/widgets/_window-picker.scss
@@ -48,6 +48,4 @@ $window_close_button_padding: 3px;
 .workspace-background {
   // keep in sync with BACKGROUND_CORNER_RADIUS_PIXELS in workspace.js
   border-radius: 30px;
-  background-color: $osd_bg_color;
-  box-shadow: 0 4px 16px 4px transparentize(darken($osd_bg_color, 30%), 0.7);
 }
-- 
GitLab

